nameable.rb

```
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
```

- [Optional] Let's raise this error like this:

   `raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"`